README.md

```diff
@@ -256,8 +261,7 @@ To create a new plugin (check the [example](plugin_examples/dummy.go) dummy plug
 
 Find the official gatherers code in: https://github.com/trento-project/agent/tree/main/internal/factsengine/gatherers
 
-
-> *** By now, it only supports Golang based implementations, but this could be extendable (if this requirement is needed, please open a Github ticket with this feature request).
+> \*\*\* By now, it only supports Golang based implementations, but this could be extendable (if this requirement is needed, please open a Github ticket with this feature request).
```

Comment:
`\*\*\*` :eyes:

Reply:
hmm it seems the auto format on save hit hard xD